internal/alloycli/cmd_fmt.go

```diff
@@ -107,6 +115,14 @@ func format(filename string, fi os.FileInfo, r io.Reader, write bool) error {
 	// Add a newline at the end of the file.
 	_, _ = buf.Write([]byte{'\n'})
 
+	// If -t/--test flag is check, only check if file is formatted correctly
+	if test {
+		if !reflect.DeepEqual(bb, buf.Bytes()) {
+			return fmt.Errorf("File %s is not formatted correctly", filename)
```

Comment:
nit: in the error msg I would add that the file can be formatted with the --write flag